### DISCUSSION TITLE
Two lines in the same window

### DISCUSSION THREAD

**Discussion by solucionesuno** [InnovateAsterisk/Browser-Phone#431] [Q&A]
Is possible to register two Lines ( sip users ) in the same session?

**Comment by InnovateAsterisk** [comment]
No, the Browser Phone cannot register multiple accounts. 

Then again, this could be achieved on the server side, a bit of cleaver dial-plan, and you could receive calls from multiple different DID's. 

Generally speaking its not often necessary to have multiple registrations when you are on the same box... If you have different account, on different servers, sorry this too is not possible. However there are a number of ways to solve this on the server side. You could use a proxy, or even just another Asterisk Box. This second Asterisk box could be started up to perform only the task of aggregating multiple SIP extensions to a single WebRTC one. Its actually a desirable solution, because calls can be held back until you are "_not on the line_", and you would have a very clear presence status. (remember presence across multiple boxes or multiple accounts can be a pain).
